guac-visualizer.md

```diff
@@ -26,90 +26,50 @@ well as a means to explore the supply chain and prototype policies.
 
 ## Requirements
 
-- [Git](https://git-scm.com/downloads)
-- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
-- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
-- A fresh copy of the [GUAC service infrastructure through Docker Compose]({{
-  site.baseurl }}{%link setup.md %}).
+- [git](https://git-scm.com/downloads)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+- [node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [GUAC services up and running]({{
+  site.baseurl }}{%link setup.md %})
 
-## Step 1: Getting started
+## Step 1. Getting started
```

Comment:
@jeffmendoza should we move this into the setup section along with the docker-compose install?